docs/modules/ROOT/pages/advanced_usage/options.adoc

```diff
@@ -10,6 +10,7 @@ The other options are:
 [options="header"]
 |===
 | Option | Description
+| `--showsettings` | Open the settings dialog while starting.
```

Comment:
Why do you only show the long form in help but dont show the short form `-s` ?
There are many examples in eg. occ commands in docs where we show both, just in the way it has been designed.

Reply:
Yes you are right, i should add the `-s`  in the docs